packages/cli/e2e/__tests__/trigger.spec.ts

```diff
@@ -50,4 +50,19 @@ describe('trigger', () => {
     expect(result.stdout).not.toContain('Staging Backend Check')
     expect(result.status).toBe(0)
   })
+
+  test('Should return code 0 when no checks match', () => {
```

Comment:
This won't pass until https://github.com/checkly/checkly-backend/pull/4139 is deployed